README.md

```diff
@@ -29,4 +29,8 @@ Then in another terminal, you can build after each change. Use the one appropria
 
 > npm run build-firefox
 
+## Semantic Versioning
+
+You can update the version of the repo with proper prefixes. For instance, prefix your commit message with `major:`, `minor:`, or `patch:` to update the version of the repository. Please be aware that updating this will cause the package to be updated in npm registry upon merging to `master`
```

Comment:
Ok, cool. So according to [this action.yml file](https://github.com/phips28/gh-action-bump-version/blob/master/action.yml#L22), the wording is as follows
```yml
  minor-wording:
    description: 'Words list that trigger a minor version bump'
    default: 'feat,minor'
    required: false
  major-wording:
    description: 'Words list that trigger a major version bump'
    default: 'BREAKING CHANGE,major'
    required: false
  patch-wording:
    description: 'Words list that trigger a patch version bump'
    required: false
```